api/src/client/dynamics/housing/DynamicsHousingRegistrationStatusClient.ts

```diff
@@ -1,24 +1,30 @@
 import {
-  HotelMotelRegistrationClient,
   HousingPropertyInterestClient,
+  HousingRegistrationClient,
   HousingRegistrationRequestLookupResponse,
 } from "@client/dynamics/housing/types";
 import { AccessTokenClient } from "@client/dynamics/types";
 import { LogWriterType } from "@libs/logWriter";
+import { PropertyInterestType } from "@shared/housing";
 
 type Config = {
   accessTokenClient: AccessTokenClient;
-  housingHotelMotelRegistrationClient: HotelMotelRegistrationClient;
+  housingHousingRegistrationClient: HousingRegistrationClient;
   housingPropertyInterestClient: HousingPropertyInterestClient;
 };
 
-export const DynamicsHotelMotelRegistrationStatusClient = (
+export const DynamicsHousingRegistrationStatusClient = (
   logWriter: LogWriterType,
```

Comment:
[dust] It doesn't look like the `logWriter` is being used here.

Reply:
Good catch, I'll remove that param